mParticle-Apple-SDK/mParticle.m

```diff
@@ -216,6 +176,8 @@ - (void)selectPlacements:(NSString *)identifier
     onEmbeddedSizeChange:(void (^ _Nullable)(NSString * _Nonnull, CGFloat))onEmbeddedSizeChange {
     NSArray<NSDictionary<NSString *, NSString *> *> *attributeMap = [self getRoktPlacementAttributes];
     
+    attributes = [self confirmSandboxAttribute:attributes];
```

Comment:
Alex pointed out to me that `sandbox` shouldn't be set on the mParticle user, since that is pretty extraneous.  Following his PR for inspiration:

https://github.com/mParticle/mparticle-web-sdk/pull/1019/files#diff-7cb37e3efb8e0c8ba283a333407dd58bed345121a89f3b82014aabf2048cc61cR145-R151